### DISCUSSION TITLE
Description on strong_product

### DISCUSSION THREAD

**Discussion by satemochi** [networkx/networkx#7509] [Q&A] [answered]
P has an edge ((u, v), (x, y)) if and only if ----

 ((u, x) (v, y)), right?

**Comment by rossbar** [comment]
I'm not sure I understand the question - can you provide a bit more info?

**Reply by satemochi** [reply]
In the following API docstring for strong_product, there is an error the definition. I think.

https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.operators.product.strong_product.html

**Reply by Schefflera-Arboricola** [reply, marked as answer]
yes, the definition is a bit inaccurate.
ref. https://en.wikipedia.org/wiki/Strong_product_of_graphs

Please feel free to make a PR or an issue on this :)

Thank you @satemochi for reporting :)

**Reply by satemochi** [reply]
Juneja-san, thank you for the reply. 

> Please feel free to make a PR or an issue on this :)

But, I have never used git commands except for add/commit/push. So... 

 I love NetworkX!

Sincerely yours!

**Reply by Schefflera-Arboricola** [reply]
> But, I have never used git commands except for add/commit/push.

It's not that complicated, you just have to fork this repo, then clone your fork on your local machine and then create a new branch(`git checkout -b new_branch_name`), and then do add - commit - push to this new branch(Also, make sure to run `pre-commit` after `git add .`). Then come back to your fork on GitHub and you will see an option to create a PR. For more details see [NetworkX's contributor guide](https://networkx.org/documentation/stable/developer/contribute.html#contributor-guide) or [this video](https://youtu.be/8lGpZkjnkt4?si=Ywf_bRAcqOaoHF1A). We would love to have you as a contributor :)

Also, I'd appreciate it if you could just address me by my username(@Schefflera-Arboricola).

Thank you :)
